packages/admin/src/routes/entities/components/ListView/ListView.js

```diff
@@ -30,8 +31,10 @@ const ListView = ({match, history, currentViewInfo, emitAction, persistViewInfo,
 
   const handleNavigateToAction = ({definition, selection}) => {
     const entityBaseUrl = goBack(match.url)
+    const searchString = selectionUtil.selectionToQueryString(selection)
```

Comment:
if you would name the var "search" you could use the property shorthand on line 37